example/src/components/BusinessInformation/index.tsx

```diff
@@ -48,6 +48,16 @@ const BusinessInformationUI = (props: BusinessInformationParams) => {
       />
       <WrapMainContent>
         <InnerContent>
+          <GrayBackground>
+            <OText size={16} weight='bold'>{t('BUSINESS_DETAILS', 'Business Details')}</OText>
+          </GrayBackground>
+          {businessState?.business?.email && (
```

Comment:
This validation will fail, please convert this validation in boolean using `!!` operator